connexion/apis/flask_api.py

```diff
@@ -165,6 +165,9 @@ def _get_flask_response_from_connexion(cls, response, mimetype):
     @classmethod
     def _build_flask_response(cls, mimetype=None, content_type=None,
                               headers=None, status_code=None, data=None):
+        # use content type specified in headers, if set
+        content_type = headers.get("Content-Type", content_type) if headers else None
```

Comment:
I would expect that passing `content_type` explicitly would take precedent over headers.
why not:
```
if content_type is None:
    content_type = headers.get("content-type")
```

Reply:
I think the headers should take precedence because it is the only way to pass such values in the tuple based controller return API. Also, the `content_type` parameter is defaulting to `mimetype` in some paths and this `mimetype` is set according to the `produces` spec parameter.

However, the `... else None` part should be `... else content_type`.